scipy/spatial/tests/test_distance.py

```diff
@@ -769,6 +770,7 @@ def test_pdist_seuclidean_random_float32(self):
         # Check no error is raise when V has float32 dtype (#11171).
         V = np.var(X, axis=0, ddof=1)
         Y_test2 = pdist(X, 'seuclidean', V=V)
+        # SHould Y_test1 be replaced by Y_test2 in the _assert_within_tol call?
         _assert_within_tol(Y_test1, Y_right, eps)
```

Comment:
@jeremiedbb Should the call to `_assert_within_tol` use `Y_test1` or `Y_test2`?

Reply:
Oh right it's a mistake it should be `Y_test2`

Reply:
Changed it.